crates/ruff_linter/src/rules/flake8_pyi/rules/docstring_in_stubs.rs

```diff
@@ -29,18 +31,41 @@ use crate::checkers::ast::Checker;
 #[violation]
 pub struct DocstringInStub;
 
-impl Violation for DocstringInStub {
+impl AlwaysFixableViolation for DocstringInStub {
     #[derive_message_formats]
     fn message(&self) -> String {
         "Docstrings should not be included in stubs".to_string()
     }
+
+    fn fix_title(&self) -> String {
+        "Remove docstring".to_string()
+    }
 }
 
 /// PYI021
-pub(crate) fn docstring_in_stubs(checker: &mut Checker, docstring: Option<&ExprStringLiteral>) {
-    if let Some(docstr) = docstring {
-        checker
-            .diagnostics
-            .push(Diagnostic::new(DocstringInStub, docstr.range()));
-    }
+pub(crate) fn docstring_in_stubs(
+    checker: &mut Checker,
+    definition: &Definition,
+    maybe_docstring: Option<&ExprStringLiteral>,
+) {
+    let Some(docstring) = maybe_docstring else {
+        return;
+    };
+    let statements = match definition {
+        Definition::Module(module) => module.python_ast,
+        Definition::Member(member) => member.body(),
+    };
+
+    let range = docstring.range();
+
+    let edit = if statements.len() == 1 {
+        Edit::range_replacement("pass".to_string(), range)
```

Comment:
In a stub file this fix will cause us to start emitting https://docs.astral.sh/ruff/rules/pass-statement-stub-body/ on the stub. You should replace the docstring with `...` instead of `pass`.